### DISCUSSION TITLE
For some reason IntelliSense doesn't work

### DISCUSSION THREAD

**Discussion by Kotsuha** [mark-wiemer/ahkpp#450] [General]
Visual Studio Code
Version: 1.91.0

Thanks for this extension.
Recently I started to use AutoHotkey v2, so I installed this extension. For some reason the IntelliSense doesn't seem to work?
For example, look at the screenshot:
![image](https://github.com/user-attachments/assets/11a274b4-3ef3-4bdd-a02e-f70168889a06)
When I type `msg`, the drop down only has "abc" items.

However, if I switch to use v1, the IntelliSense works.

![image](https://github.com/user-attachments/assets/6b170cdc-a846-474e-81d5-4f8937c40bc7)

Am I missing something?

**Comment by fade2gray** [comment]
V2 Intellisense in this extension is still 'work in progress' - you may want to install 'AutoHotkey v2 Language Support' by thqby, although that extension does not support V1.

**Comment by Kotsuha** [comment]
Thanks for letting me know. Should I close this issue?

**Comment by fade2gray** [comment]
Heads-up, after the fact, but I failed to mention that @mark-wiemer, the author of this extension, released a new 'AutoHotkey Extension Pack' containing ...

- AutoHotkey Plus Plus
- AutoHotkey v2 Language Support
- vscode-autohotkey-debug

... see https://github.com/mark-wiemer-org/ahkpp/discussions/21#discussion-1310547 - makes it a little easier to manage the three extensions.

But I'd leave your comment up in case anyone else should have the same query.
